src/scripts/charts-screenshots.ts

```diff
@@ -0,0 +1,100 @@
+import 'dotenv';
+
+import fs from 'fs';
+// import path from 'path';
+import puppeteer, { Page } from 'puppeteer';
+
+import chartUrls from './constants/chart-urls';
+
+// const iPad = puppeteer.devices['iPad landscape'];
+
+function delay(time: number) {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve('success');
+    }, time);
+  });
+}
+
+const FRONTEND_SITE_URL = (
+  process.env.ALLOWED_ORIGINS || 'https://explorer.pastel.network'
+).split(',')[0];
+
+function getFileSize(base64String: string) {
+  const stringLength = base64String.length - 'data:image/png;base64,'.length;
+
+  const sizeInBytes =
+    (4 * Math.ceil(stringLength / 3) * 0.5624896334383812) / 1024;
```

Comment:
Very suspicious constant, are you sure it's correct a way?

Reply:
I updated
This commit:
https://github.com/pastelnetwork/pastel-explorer-backend/pull/45/commits/f4b9e3a8a8bd58e4c57214464e70d6cab3945610